Core/Core/Serialisation/BaseObjectSerialzerUtilities.cs

```diff
@@ -17,194 +17,47 @@
 
 namespace Speckle.Core.Serialisation;
 
-internal static class SerializationUtilities
+internal static class CallSiteCache
```

Comment:
Could we just split these classes into their own files? The diff is madness now 🤣

Reply:
Lets split it into individual files, it's easier to maintain in the future